src/averageAges.js

```diff
@@ -20,6 +20,18 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const menFilteredByCentury = century
+    ? men.filter(person => Math.ceil(person.died / 100) === century)
+    : men;
+
+  const sumOfMenAges = menFilteredByCentury.reduce((sum, man) =>
+    sum + (man.died - man.born), 0);
```

Comment:
you use this logic a couple of times in the task, so it seems reasonable to create a helper function for that